tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -4320,60 +4332,61 @@ def closeLoop(self, kernel, tPA, tPB, loopIdx, finalLoop, emitEndLabelOnly=False
       else:
         endCounter = 0
 
-      if kernel["AssertSummationElementMultiple"] % (kernel["DepthU"] * 2) == 0 and endCounter > 0:
```

Comment:
This segment is not related to dot2 kernel, but it leads to runtime error due to undefined variable **decValue**. Should we modify or remove this?